Flow.Launcher/Helper/ErrorReporting.cs

```diff
@@ -4,45 +4,52 @@
 using Flow.Launcher.Infrastructure;
 using Flow.Launcher.Infrastructure.Exception;
 
-namespace Flow.Launcher.Helper
+namespace Flow.Launcher.Helper;
+
+public static class ErrorReporting
 {
-    public static class ErrorReporting
+    private static void Report(Exception e)
+    {
+        var logger = LogManager.GetLogger("UnHandledException");
+        logger.Fatal(ExceptionFormatter.FormatExcpetion(e));
+        var reportWindow = new ReportWindow(e);
+        reportWindow.Show();
+    }
+
+    public static void UnhandledExceptionHandle(object sender, UnhandledExceptionEventArgs e)
     {
-        private static void Report(Exception e)
-        {
-            var logger = LogManager.GetLogger("UnHandledException");
-            logger.Fatal(ExceptionFormatter.FormatExcpetion(e));
-            var reportWindow = new ReportWindow(e);
-            reportWindow.Show();
-        }
-
-        public static void UnhandledExceptionHandle(object sender, UnhandledExceptionEventArgs e)
-        {
-            //handle non-ui thread exceptions
-            Report((Exception)e.ExceptionObject);
-        }
-
-        public static void DispatcherUnhandledException(object sender, DispatcherUnhandledExceptionEventArgs e)
-        {
-            //handle ui thread exceptions
-            Report(e.Exception);
-            //prevent application exist, so the user can copy prompted error info
-            e.Handled = true;
-        }
-        
-        public static string RuntimeInfo()
-        {
-            var info = $"\nFlow Launcher version: {Constant.Version}" +
-                       $"\nOS Version: {ExceptionFormatter.GetWindowsFullVersionFromRegistry()}" +
-                       $"\nIntPtr Length: {IntPtr.Size}" +
-                       $"\nx64: {Environment.Is64BitOperatingSystem}";
-            return info;
-        }
-
-        public static string DependenciesInfo()
-        {
-            var info = $"\nPython Path: {Constant.PythonPath}\nNode Path: {Constant.NodePath}";
-            return info;
-        }
+        //handle non-ui thread exceptions
+        Report((Exception)e.ExceptionObject);
+    }
+
+    public static void DispatcherUnhandledException(object sender, DispatcherUnhandledExceptionEventArgs e)
+    {
+        //handle ui thread exceptions
+        Report(e.Exception);
+        //prevent application exist, so the user can copy prompted error info
+        e.Handled = true;
+    }
+
+    public static string RuntimeInfo()
+    {
+        var info =
+            $"""
+
+             Flow Launcher version: {Constant.Version}
+             OS Version: {ExceptionFormatter.GetWindowsFullVersionFromRegistry()}
+             IntPtr Length: {IntPtr.Size}
+             x64: {Environment.Is64BitOperatingSystem}
+             """;
```

Comment:
I wonder whether this will make the string ends with a newline?

Reply:
It will preserve the new line I added after opening the string and it will ignore the rest of unnecessary whitespace as per [documentation](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/raw-string).
![image](https://github.com/Flow-Launcher/Flow.Launcher/assets/3993179/2834fdab-4db6-4a41-8cd0-71e6b18aded5)

Reply:
I think regardless it is better than before. We can still use `\n` if you think that the first empty line feels weird?

Reply:
I do think it looks a little bit weird, but the alternative is this, because you can't have escape sequences in raw strings:

```csharp
        var info =
            $"""
             {"\n"}Flow Launcher version: {Constant.Version}
             OS Version: {ExceptionFormatter.GetWindowsFullVersionFromRegistry()}
             IntPtr Length: {IntPtr.Size}
             x64: {Environment.Is64BitOperatingSystem}
             """;
```

Reply:
Or something like this:

```csharp
        var info = "\n" +
            $"""
             Flow Launcher version: {Constant.Version}
             OS Version: {ExceptionFormatter.GetWindowsFullVersionFromRegistry()}
             IntPtr Length: {IntPtr.Size}
             x64: {Environment.Is64BitOperatingSystem}
             """;
```

The second example looks better to me than the first one, but I'm still not sure if it looks better than just having it in the raw string directly.